modules/data-entry/src/main/frontend/src/questionnaireEditor/Question-hints.json

```
@@ -0,0 +1,6 @@
+{
+  "entryMode": "Specifies the source of the value for this question: manually entered by the **user**, **computed** using a specified expression, or copied over from another **reference** answer to a question specified by a path.",
+  "expression": "Used with the **computed** entry mode. Supports javascript syntax.\n\nExample:\n\n```return 2 * @{age:-0}```\n\n where `age` is the id of a question in the same form, `:-` is an optional marker for a default value, and `0` is the optional default value.",
```

I would remove "Used with the computed entry mode", this only shows up if the user already selected that. Same for reference below.

```suggestion
  "expression": "Supports javascript syntax, with special placeholder variables being populated with answers to other questions in the form.\n\nExample:\n\n```return @{laps} * @{lapLength:-100}```\n\n where `laps` and `lapLength` are the names of questions in the same form, `:-` is an optional marker for a default value, and `100` is the optional default value.",
```